lib/common.sh

```diff
@@ -184,6 +184,7 @@ export DEFAULT_HOSTS_MEMORY=${DEFAULT_HOSTS_MEMORY:-4096}
 export CLUSTER_NAME=${CLUSTER_NAME:-"test1"}
 export CLUSTER_APIENDPOINT_IP=${CLUSTER_APIENDPOINT_IP:-"192.168.111.249"}
 export KUBERNETES_VERSION=${KUBERNETES_VERSION:-"v1.21.0"}
+export UPGRADED_K8S_VERSION=${UPGRADED_K8S_VERSION:-"v1.21.1"}
```

Comment:
why do we need this here?

Reply:
Please check the above comment. We need it define it globally here, so all dependent tests(upgrade, scale-in && node reuse) on this var can use it.